pallets/handles/src/handles-utils/src/suffix.rs

```diff
@@ -33,15 +33,15 @@ pub fn generate_unique_suffixes(
 	canonical_base: &str,
 ) -> impl Iterator<Item = u16> + '_ {
 	let seed = generate_seed(canonical_base);
-	let mut rng = SmallRng::seed_from_u64(seed);
+	let mut rng = Rand32::new(seed);
 
 	let mut indices: Vec<u16> = (min..=max).collect();
-	let min = min as usize;
-	let max = max as usize;
+	// let min = min as usize;
+	// let max = max as usize;
 	(min..=max).rev().map(move |i| {
-		let j = rng.gen_range(min..=i);
-		indices.swap(i - min, j - min);
-		indices[i - min]
+		let j = rng.rand_range((min as u32)..(i + 1) as u32) as u16;
```

Comment:
why do do as u32?

Reply:
Because it requires u32 as args and only outputs u32

Reply:
It would be nice if the crate had a Rand16 but it doesn't.

Reply:
Do you mind adding in the commit why oorandom was chosen?